packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-app-in-okta/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Create an App in Okta
+---
+You need to create an OpenID Connect app in your Okta org. This app consumes the response from the Identity Provider after authentication and authorization, allowing user access to your application. Users that sign in for the first time are created in Okta and are associated with this application.
```

Comment:
We should probably also address users who _don't_ need to create an app right now.

If I already have an app that represents the software I'm building, or already have a test app because I've already added one provider, what should I do? Do I need to create an additional app?

cc @ZoeMonosson-okta

Reply:
Good call, cuz I used an existing one and most often others would, too.